README.md

```diff
@@ -44,6 +44,77 @@ This is the basic outline of the project and is a work in progress.
 
 ## Usage Examples
 
+# Public auth example for RequestHandler
+```php
+$config [
+    'key' => CONSUMER_KEY,
+    'secret' => CONSUMER_SECRET,
+    'sign_with' => 'HMAC-SHA1',
+    // other options
+]
+
+// Get Request Token
+$handler = new RequestHandler($config);
+try {
+    $handler->request('GET', 'Currencies');
+} catch (AuthException $e) {
+    $authData = $handler->getAuthData();
+    // we store oauth_token from $authData and direct user to
+    // https://api.xero.com/oauth/Authorize?oauth_token=REQUEST_TOKEN_HERE
+}
+
+// Get Access Token
+// We assume that user granted access by visiting this url
+// https://app.xero.com/oauth/APIAuthorise?oauth_token=REQUEST_TOKEN_HERE
+$config['token'] = REQUEST_TOKEN_HERE;
+$config['token_secret'] = $authData['oauth_token_secret']; // see above
+$config['oauth_verifier'] = '563499'; // we collect after user grants permission
+$handler = new RequestHandler($config);
+$handler->request('GET', 'Currencies');
+$authData = $handler->getAuthData();
```

Comment:
It's for host application.
Eventually all these options should be passed from main class (not directly into RequestHandler).

For private auth there are no such problems (just pass correct private key and voila) but here we should get request token, wait for user to authorize, get verifier from user's authorization callback and finally get access token. I do not see way to make it automatic.
@darrynten